app/src/main/java/woowacourse/paint/model/DrawingElement.kt

```diff
@@ -0,0 +1,26 @@
+package woowacourse.paint.model
+
+import android.graphics.Paint
+import android.graphics.Path
+
+data class DrawingElement(
+    val path: Path = Path(),
+    val paint: Paint = Paint(),
+) {
+
+    fun movePath(x: Float, y: Float) {
+        path.moveTo(x, y)
+    }
+
+    fun initPath(x: Float, y: Float) {
+        path.lineTo(x, y)
+    }
+
+    fun setStroke(value: Float) = this.copy(path = Path(), paint = paint).apply {
```

Comment:
```suggestion
    var stroke: Float
        get() = paint.strokeWidth
        set(value) { paint.strokeWidth = value }
```
불변 객체를 포기한다면 다음과 같이 setter를 통해서 밖에서 사용할 때 좀 더 깔끔한 문법을 작성해볼 수 있습니다!

Reply:
오오 이렇게 깔끔하게 작성할 수도 있네요👍
이렇게 제안해 주신 부분은 나중에 불변객체를 사용하지 않을 때 사용해보겠습니다😊